database/changesets/20210422_TerraDataRepoSourceTable.xml

```diff
@@ -16,13 +16,13 @@
             the 'Terra DataRepo' source table schema
         </comment>
         <sql>
-            CREATE TYPE TerraDataRepoSource AS (
-                id              bigint,      -- unique to this table
-                dataset         text,        -- the id of the TDR dataset
-                dataset_table   text,        -- the name of the dataset table
-                snapshot        text,        -- name of a column in table containing orderable and strictly increasing data
-                details         text,        -- the identifier of the source details table
```

Comment:
Did you mean to eliminate `details`?

Reply:
AH I didn't mean to 🤦  thanks for catching it!